COLLABORATOR_GUIDE.md

```diff
@@ -403,10 +403,11 @@ recommended but not required.
 
 ### Deprecations
 
-_Deprecation_ refers to the identification of Public APIs that should no longer
-be used and that may be removed or modified in backward-incompatible ways in
-a future major release of Node.js. Deprecation may be used with internal APIs if
-there is expected impact on the user community.
+__Deprecation__ refers to the identification of Public APIs that should no
+longer be used.
+
+Once specific bits of code are marked as being Deprecated, they are no longer
+covered by the typical current or Long Term Support policies.
```

Comment:
Remove `typical`. More importantly, what does this mean? Is this all about semver or is it something else?

Reply:
For instance, a deprecated function in an LTS branch would not fall under the same scrutiny and care as a non-deprecated function in that same branch, meaning we may be more liberal about what changes we allow to be made within those

Reply:
Can we link to these policies here?